presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotClusterInfoFetcher.java

```diff
@@ -486,4 +504,98 @@ public TimeBoundary getTimeBoundaryForTable(String table)
             throw e;
         }
     }
+
+    public static class Instance
+    {
+        private final String instanceName;
+        private final String hostName;
+        private final boolean enabled;
+        private final int port;
+        private final int grpcPort;
+        private final List<String> tags;
+        private final List<String> pools;
+
+        @JsonCreator
+        public Instance(
+                @JsonProperty String instanceName,
+                @JsonProperty String hostName,
+                @JsonProperty boolean enabled,
+                @JsonProperty int port,
+                @JsonProperty int grpcPort,
+                @JsonProperty List<String> tags,
+                @JsonProperty List<String> pools)
+        {
+            this.instanceName = instanceName;
+            this.hostName = hostName;
+            this.enabled = enabled;
+            this.port = port;
+            this.grpcPort = grpcPort;
+            this.tags = tags;
+            this.pools = pools;
+        }
+
+        public String getInstanceName()
```

Comment:
Is this class json serializable? If it is, let's annotate each getter with `@JsonProperty`.